Scionlab revocation info

FR4NK-W opened the pull request:
Cherry pick commit 9d88217

Only take 'bad RevocationInfo messages' fix

@jonghoonkwon
Reopened from https://github.com/scionproto/scion/pull/1343

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/netsec-ethz/netsec-scion/2)
<!-- Reviewable:end -->

jonghoonkwon commented:
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[python/beacon_server/base.py, line 299 at r1](https://reviewable.io:443/reviews/netsec-ethz/netsec-scion/2#-L-g4g3_34uPHDsjrlAy:-L-g4g3aBxp-qpn3NWWB:bmvmza8) ([raw file](https://github.com/netsec-ethz/netsec-scion/blob/2ae75c209c2d278b8b289744267bede0d42c1dd7/python/beacon_server/base.py#L299)):*
> ```Python
>         this function gets called to continue the processing for the pcb.
>         """
>         pseg = seg_meta.seg
> ```

Somehow, the new variable name may bring a bit confusion here, unless we update all the variable name for the 'pcb' in the scion codebase. 
Lets keep the variable name 'pcb' for now.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/netsec-scion/2)*
<!-- Sent from Reviewable.io -->

FR4NK-W commented:
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[python/beacon_server/base.py, line 299 at r1](https://reviewable.io:443/reviews/netsec-ethz/netsec-scion/2#-L-g4g3_34uPHDsjrlAy:-L-jsKcfK-8bfcnE6uUe:b-px9gk5) ([raw file](https://github.com/netsec-ethz/netsec-scion/blob/2ae75c209c2d278b8b289744267bede0d42c1dd7/python/beacon_server/base.py#L299)):*
<details><summary><i>Previously, jonghoonkwon wrote…</i></summary><blockquote>

Somehow, the new variable name may bring a bit confusion here, unless we update all the variable name for the 'pcb' in the scion codebase. 
Lets keep the variable name 'pcb' for now.
</blockquote></details>

Done.
Removed all changes not related to exception handling, removed variable renaming and update error messages.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/netsec-scion/2)*
<!-- Sent from Reviewable.io -->

jonghoonkwon commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/netsec-scion/2#-:-L-kUoHg0VnJINIUXFTY:bnfp4nl)*
<!-- Sent from Reviewable.io -->

matzf commented:
Closing this ancient PR.
